netmaster/master/netmaster.go

```diff
@@ -175,6 +185,30 @@ func startServiceContainer(tenantName string) error {
 	return err
 }
 
+func stopServiceContainer(tenantName string) error {
```

Comment:
minor nit, may be this should be called `stopAndRemoveServiceContainer` to indicate the fact that it also destroys the service container.